### PR TITLE
Bugfix/tb bugfix 0102

### PR DESCRIPTION
**Pull request by tb365**
**这个 PR 实现什么功能/修复什么问题**:
* elastic skus add region_ext_id & zone_ext_id 
* disk delete condition validator 
* huawei oss client read http ProxyFromEnvironment 
* network operations add action log
* account create bugfix

**是否需要 backport 到之前的 release 分支**:
- release/2.13
- release/2.14
- release/3.0

/area region
/cc @swordqiu @yousong @ioito

### CODE REVIEW

File: pkg/compute/models/cloudaccounts.go [hunk 1]
```diff
@@ -142,6 +142,18 @@ func (self *SCloudaccount) GetCloudproviders() []SCloudprovider {
 	return self.getCloudprovidersInternal(tristate.None)
 }
 
+func (self *SCloudaccount) IsAvailable() bool {
+	if !self.Enabled {
+		return false
+	}
+
+	if self.HealthStatus != api.CLOUD_PROVIDER_HEALTH_NORMAL {
```
Comment (ioito) [inline pkg/compute/models/cloudaccounts.go]: 这里应该是!utils.IsInStringArray(self.healthStatus, api.CLOUD_PROVIDER_VALID_HEALTH_STATUS)

Comment (tb365) [reply pkg/compute/models/cloudaccounts.go]: fix

**Pull request merged**